cricketscircle/jekyll-test
# Formatting guide

## How to test

Visit http://127.0.0.1:4000/jekyll-test/formatting-guide/ and you should see the formatting guide.
